ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdated.java

```diff
@@ -154,7 +155,7 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
           "Invalid payload attributes: {}",
           () ->
               maybePayloadAttributes.map(EnginePayloadAttributesParameter::serialize).orElse(null));
-      return new JsonRpcErrorResponse(requestId, JsonRpcError.INVALID_PAYLOAD_ATTRIBUTES);
```

Comment:
I didn't see this change mentioned in https://github.com/ethereum/execution-apis/pull/338 - is this something different?

Reply:
Yea, bring some changes from https://github.com/ethereum/execution-apis/pull/337 too. Sorry should have updated the description.

Reply:
my bad, I got Zhenyang to crowbar that change in too :)

Reply:
ok makes sense